### PR TITLE
put temporary message about new branch structure in README.md

### PR DESCRIPTION
**Pull request by GodloveD**
**Description of the Pull Request (PR):**

Put a temporary message about our new branch structure in the README.md.  This is against master.

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [ ] I have tested this PR locally with a `make test`
- [ ] This PR is NOT against the project's master branch
- [x] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+_Please note recent changes in the github repo branch structure.  If you want to
+install a stable release if Singularity, please use a tag or a [release
```
Comment (jtriley) [inline README.md]: "release of"

Comment (GodloveD) [reply README.md]: Thank you sir!

**Review by dtrudg — APPROVED** [review]
Looks good to me.

**Comment by GodloveD** [comment]
I'm just going to go ahead and merge this since it is time sensitive and has had 2 sets of eyes on it already.  Thanks @jtriley and @dctrud!

**Pull request merged**